src/highlight/highlight.ts

```diff
@@ -1,7 +1,13 @@
-const get = require('lodash/get');
 import { Component, Input } from '@angular/core';
 import { bem } from '../utils';
 
+function getPropertyByPath(object: object, path: string): any {
```

Comment:
Why not take the exact same [implementation](https://github.com/algolia/instantsearch.js/blob/9296022fecadfbf82f15e837c215a1356eac4bc5/src/lib/utils/getPropertyByPath.ts) used in IS.js and RIS?

Reply:
it's the implementation of React InstantSearch, which supports bracket-notation ([0])